### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by pYr0x**
update to generator-donejs 0.5.0

also updated the tests, so they are pass with the new generator version.
main change is, that there is a .yo-rc.json and a package.json in the test folder.
at the beginning of the test i am [changing the working directory](https://github.com/pYr0x/cli/blob/6076d46e352362c2e6e314f12f968077e3624a38/test/utils.js#L14-L17) so yeoman is finding the root folder by searching for .yo-rs.json file. at the same folder the package.json is located.
generator-donejs will find the package.json because it uses the yeoman function https://github.com/pYr0x/generator-donejs/blob/60-use-package-json/component/index.js#L65

this replaces #35

### CODE REVIEW

**Comment by matthewp** [comment]
Cool, when the version is released i'll rerun these tests.

**Comment by matthewp** [comment]
@pYr0x btw, you are on the donejs core team so you should be able to push to branches on this repo instead of having to use your fork.  Same for generator-donejs

**Comment by matthewp** [comment]
Ok, checks out.

File: test/.yo-rc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (daffl) [inline test/.yo-rc.json]: I'm fairly certain this file can be removed now.

Comment (pYr0x) [inline test/.yo-rc.json]: i think yoeman need that file to identify that this is the root folder

**Pull request merged**